src/github.com/getlantern/flashlight/main/main.go

```diff
@@ -266,7 +265,6 @@ func afterStart(cfg *config.Config) {
 
 func onConfigUpdate(cfg *config.Config) {
 	autoupdate.Configure(cfg)
-	proxiedsites.Configure(cfg.ProxiedSites)
```

Comment:
Oh right! that's true, I only meant to disable the ui service but this change is definitely wrong.